setup/pi/configure.sh

```diff
@@ -146,6 +146,7 @@ function install_archive_scripts () {
 
     log_progress "Installing base archive scripts into $install_path"
     get_script $install_path archiveloop run
+    get_script $install_path tesla_api.py run
```

Comment:
How about doing this only when the tesla api variables are set, and then in archive-clips.sh, check whether /root/bin/tesla_api.py exists before calling it? That would make it truly optional, and wouldn't install the script unnecessarily.

Reply:
How would I go about checking if the variables are set in `configure.sh`?

Reply:
Something like:
```
if [ ! -z ${tesla_email:+x} ]
then
  get_script $install_path tesla_api.py run
else
  log_progress "Skipping tesla_api install"
fi
```